scripts/cfos/tests/README.md

```diff
@@ -1,18 +1,24 @@
 ## Test schema file setup ##
-You can either:
-- replace the contents of 'scripts/cfos/dataset_tables_schema.json' file with the contents of the 'scripts/cfos/tests/test_schema.json' file temporarily OR
-- change the code in make_dataset_data_tables.py so that schema_json points to the test schema file instead of the real one
+You can run the code using the test schema by adding the parameter "-t True" to run in testing mode (examples shown below in How to Run section)
 
 ## How to Run ##
 ### test fail ### 
-python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Fail.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing
+python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Fail.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing -t True
 
 ### test pass ###
-python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Pass.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing
+python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Pass.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing -t True
 
 
 ## Validating outcomes ##
 Check the validation_erros.csv and make sure expected outcomes are met. 
 General expected outcomes are explained above the column ID in the Fail.xlsx file. 
 
+The code can fail in the following ways:
+- **primary key column contains non-unique values**: 'validation_error':{row: 6, column: "id_unique"}: "X1234B" contains values that are not unique
+- **value doesn't match an expected pattern** : 'validation_error':{row: 6, column: "file_path_explicit_pattern"}: "gs://456" does not match the pattern "^fs://"
+- **value is an unexpected type**. If "number", it must be a float. If "integer", it must be an integer: 'validation_error':The column integer_only_field has a dtype of object which is not a subclass of the required type <class 'int'>
```

Comment:
is there a reason this one is formatted differently from the others? the others have a row and column identification before the message (which is useful for the user to identify which data row the offending value came from.

Reply:
I'm not sure why this is different formatting. They're all the default error messages from Pandas_schema.